api/grpcserver/v2alpha1/activation_test.go

```diff
@@ -0,0 +1,254 @@
+package v2alpha1
+
+import (
+	"context"
+	"errors"
+	"io"
+	"testing"
+	"time"
+
+	spacemeshv2alpha1 "github.com/spacemeshos/api/release/go/spacemesh/v2alpha1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/spacemeshos/go-spacemesh/common/fixture"
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/events"
+	"github.com/spacemeshos/go-spacemesh/sql"
+	"github.com/spacemeshos/go-spacemesh/sql/atxs"
+)
+
+func TestActivationService_List(t *testing.T) {
+	db := sql.InMemory()
+
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	gen := fixture.NewAtxsGenerator()
+	activations := make([]types.VerifiedActivationTx, 100)
+	require.NoError(t, db.WithTx(ctx, func(dtx *sql.Tx) error {
+		for i := range activations {
+			atx := gen.Next()
+			require.NoError(t, atxs.Add(dtx, atx))
+			activations[i] = *atx
+		}
+		return nil
+	}))
```

Comment:
Do you need a transaction here?
```suggestion
	for i := range activations {
		atx := gen.Next()
		require.NoError(t, atxs.Add(db, atx))
		activations[i] = *atx
	}
```